### PR TITLE
[R4R] Fix missing context parameter in proxy

### PR DESCRIPTION
**Pull request by yutianwu**
Ref: https://github.com/tendermint/tendermint/issues/3669

* [x] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by melekes** [comment]
Thanks for contributing to Tendermint 👍

**Comment by yutianwu** [comment]
> Thanks for contributing to Tendermint 👍

never mind :)

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -59,3 +59,4 @@
   * `Mempool#Update` and `BlockExecutor#Commit` now accept
     `[]*abci.ResponseDeliverTx` - list of `DeliverTx` responses, which should
     match `block.Txs`
+- [rpc] \#3669 Handlers in proxy routes miss context parameter
```
Comment (Liamsi) [inline CHANGELOG_PENDING.md]:
```suggestion
- [rpc] \#3669 Add context parameter to Handlers in proxy routes (@yutianwu)
```

**Comment by Liamsi** [comment]
Thanks! We should add integration tests that would have caught this.

File: CHANGELOG_PENDING.md (hunk 1 above)
Comment (yutianwu) [reply CHANGELOG_PENDING.md]: thx, updated

**Pull request merged**